src/commands/postDeploy/validateWebSite.ts

```diff
@@ -60,30 +39,28 @@ export async function validateWebSite(deploymentCorrelationId: string, siteTreeI
         let pollingIntervalMs = initialPollingIntervalMs;
         const start = Date.now();
         const uri = siteTreeItem.root.client.defaultHostUrl;
-        const options: {} = {
-            method: 'GET',
-            uri: uri,
-            resolveWithFullResponse: true
-        };
         let currentStatusCode: number | undefined = 0;
         const statusCodes: { code: number | undefined, elapsed: number }[] = [];
 
         // tslint:disable-next-line:no-constant-condition
         while (true) {
+            if (tokenSource.token.isCancellationRequested) {
+                // the user cancelled the check by deploying again
+                context.telemetry.properties.canceled = 'true';
```

Comment:
Yeah, but Stephen had this being tracked here https://github.com/microsoft/vscode-azureappservice/pull/1122/files#diff-be3a5553e0be71663344e72f967dde08L82-L86 so I didn't want to change the properties.  Thought it might mess with his data visualizations.

Reply:
Yeah but you changed it to be `throw new UserCancelledError();`, which affects the telemetry, which means you should update the telemetry and any queries/visualizations we have

Or else change it back to `break;` like it used to be